PKHeX.WinForms/Resources/text/lang_es.txt

```diff
@@ -893,32 +893,32 @@ SAV_PokedexSM.GB_Encountered=Visto
 SAV_PokedexSM.GB_Language=Idiomas
 SAV_PokedexSM.GB_Owned=Obtenido
 SAV_PokedexSM.L_goto=Ir a:
-SAV_PokedexSWSH.B_Cancel=Cancel
-SAV_PokedexSWSH.B_GiveAll=Check All
-SAV_PokedexSWSH.B_Modify=Modify...
-SAV_PokedexSWSH.B_Save=Save
-SAV_PokedexSWSH.CHK_Caught=Owned
-SAV_PokedexSWSH.CHK_G=Gigantamax
+SAV_PokedexSWSH.B_Cancel=Cancelar
+SAV_PokedexSWSH.B_GiveAll=Marcar todos
+SAV_PokedexSWSH.B_Modify=Modificar...
+SAV_PokedexSWSH.B_Save=Gaurdar
+SAV_PokedexSWSH.CHK_Caught=Capturado
+SAV_PokedexSWSH.CHK_G=Gigamax
 SAV_PokedexSWSH.CHK_Gigantamaxed=Gigantamaxed
```

Comment:
I _believe_ this should be `Gigamaxizó`, considering the official translation for the verb is gigamaxizar.

Reply:
I believe too, I couldn't check back when I posted this several hours ago, thanks either way!